\documentclass[a4paper,12pt]{moderncv}
\moderncvstyle{banking}
\moderncvcolor{blue}
\usepackage[left=60pt,top=60pt,right=60pt,bottom=60pt]{geometry}
\usepackage[inline]{enumitem}

\usepackage[T1]{fontenc}
%\usepackage{gfsdidot}

\setlength{\hintscolumnwidth}{0pt}
\linespread{1.1}

\firstname{Dr Pierre-Yves}
\familyname{Dupont}
\address{Massey University - Institute of Fundamental Sciences}{Palmerston North, Manawatu}{New Zealand - 4442}
\phone{+64-6-356 9099 X85734}
\email{p.y.dupont@massey.ac.nz}
\homepage {http://pydupont.github.io/CV/}

\begin{document}
\maketitle

\section{PERSONAL}
\begin{description}[nosep,labelwidth=1.25in,labelsep=3ex,leftmargin=!,align=right]
\item[Date of Birth] 24/4/1984

\item[Nationality] French

\item[Languages] French (mother tongue), English (advanced)

\end{description}

\section{EDUCATION}
\cvitem{2011}{\textbf{PhD in Computational Biology}, INRA, Clermont Ferrand}
\cvitem{2008}{\textbf{Masters in Computational Biology}, Bordeaux University, \emph{Class Major}}
\cvitem{2007}{\textbf{Bachelor in Computer Sciences}, Bordeaux University}
\cvitem{2005}{\textbf{Bachelor in Biology, Geology and Earth Sciences}, Bordeaux University, Diploma to become High School biology and geology teacher}

\section{POSITIONS}
\cvitem{}{\textbf{Postdoctoral researcher in computational biology at Massey University}, Palmerston North, New Zealand; 2012 -- current\\
Lateral gene transfer in fungi: Identifying the cross-species toolbox for metabolic innovation.\\}

\cvitem{}{\textbf{PhD studentship at INRA}, Clermont Ferrand, France; 2008 -- 2011\\
Thesis on developing bioinformatics tools to combine information from metabolomics, transcriptomics and promotor studies to characterize metabolic pathways involved in liver cancers.This was in collaboration with the Polytechnic school (Palaiseau, France) and a IT startup company, Solusciences (Clermont-Ferrand, France) to develop software for biological pathway modelling and biochemical dynamics analysis.\\}

\cvitem{}{\textbf{Masters Internship at European Institute of Biology and Chemistry}, Bordeaux, France; 2008\\
Evaluation of a docking software (Autodock v.4) and development of an automatic procedure to compute docking of multiple ligands on kinase receptors using a computation cluster.\\}

\cvitem{}{\textbf{Bachelor Internship at Bordeaux University}, Bordeaux, France; 2007\\
Development of software for visualization and detection of internal movements in biomolecules.\\}

\section{SKILLS}
\cvitem{Phylogeny}{Neighbor Joining, Maximum Likelyhood (PHYLIP, PhyML, RAxML), super trees, consensus trees. Multiple sequence alignments with mafft, clustal, kalign, muscle, etc.}
\cvitem{Statistics}{Good understanding of basic statistics, Markov chains and Monte Carlo statistics}
\cvitem{Computer Sciences}{Daily use of Windows and Linux (Fedora, Ubuntu, Mint) and command line interface. Good commad of languages Ruby (and Ruby on Rails), Python and Java, and web technologies (HTML 5, CSS3, JQuery). Common practice of C++, R (BioConductor) and Perl (BioPerl, EnsEMBL libraries). Good understanding of Agent-Based Models (Repast symphony) and Petri Nets}
\cvitem{Protein structure analysis and visualisation}{AutoDock, GOLD, VMD}

\section{PUBLICATIONS}

\cvitem{}{\textit{Fungal endophyte infection of ryegrass reprogrammes host metabolism and alters development}\\
\textbf{Dupont PY}, Eaton CJ, Wargent JJ, Fechtner S, Solomon P, Schmid J, Day RC, Scott B, Cox MP; New Phytologist; \textit{In press}}

\cvitem{}{\textit{A core gene set describes the molecular basis of mutualism and antagonism in Epichlo\"e spp.} \\
Eaton CJ, \textbf{Dupont PY}, Solomon P, Clayton W, Scott B, Cox MP; Molecular Plant-Microbe Interactions, 28(3), 218â€“231; PubMed PMID: 25496592}

\cvitem{}{\textit{HyLiTE: accurate and flexible analysis of gene expression in allopolyploid species}\\
Duchemin W, \textbf{Dupont PY}, Campbell MA, Ganley A and Cox MP; BMC Bioinformatics, 2015; 16:8; PubMed PMID: 25592117}
 
\cvitem{}{\textit{Chapter 10 - Genomes of plant-associated clavicipitaceae}\\
Schardl CL, Young CA, Moore N, Krom N, \textbf{Dupont PY}, Pan J, Farman ML; Advances in Botanical Research, 70, 291-327, 2014; DOI: 10.1016/B978-0-12-397940-7.00010-0}

\cvitem{}{\textit{Computational identification of transcriptionally co-regulated genes, validation with the four ANT isoform genes}\\
\textbf{Dupont PY}, Guttin A, Issartel JP, Stepien G; BMC Genomics, 13:482, 9/2012; PubMed PMID: 22978616}

\cvitem{}{\textit{Computational analysis of the transcriptional regulation of the adenine nucleotide translocator isoform 4 gene and its role in spermatozoid glycolytic metabolism}\\
\textbf{Dupont PY}, Stepien G; Gene, 487(1):38-45, 11/2011; PubMed PMID: 21827840}

\cvitem{}{\textit{Description and assessment of a model for GSK-3beta database virtual screening}\\
Ventimila N, \textbf{Dupont PY}, Laguerre M, Dessolin J; J Enzyme Inhib Med Chem, 25(2):152-7, 4/2010; PubMed PMID: 20170337}

\section{CONFERENCE}
\cvitem{}{\textit{Endophyte infection of ryegrass alters metabolism, development and response to stress} - \textit{Pierre-Yves Dupont}; Queenstown Molecular Biology Conference, 8/2014; Presented in both Plant and Main meetings}
\cvitem{}{\textit{Fungal endophyte infection of ryegrass reprograms host metabolism and alters development} - \textit{Pierre-Yves Dupont}; Fungal Genetics Conference, 3/2015; Poster}

\newpage

\section{SOFTWARE AND DATABASES}
\cvitem{}{\textbf{Genomic annotation of \emph{Epichloe festucae} E2368 gene models}}
\cvitem{}{Publically available at: \textit{\href{}{http://epichloe.massey.ac.nz}}. This database contains functional annotations predicted by sequence homology for all \emph{E. festucae} E2368 gene models. It includes links to major international bioinformatics databases including Pfam, Gene Ontology (GO), KEGG, Interproscan and GenBank.}
\cvitem{}{\textbf{Genomic annotation of perennial ryegrass (\emph{Lolium perenne}) gene models}}
\cvitem{}{Publically available at: \textit{\href{}{http://ryegrass.massey.ac.nz}}. This database contains functional annotations predicted by sequence homology for over 50,000 predicted perennial ryegrass coding sequences. It includes links to the major international bioinformatics databases listed above.}
\cvitem{}{\textbf{Hybrid Lineage Transcriptome Explorer (HyLiTE)}}
\cvitem{}{Publically available at: \textit{\href{}{http://sourceforge.net/projects/hylite}}. HyLiTE is a software allowing the analysis of high-throughput transcriptome data from allopolyploid species}


\end{document}